pkg/controller/account/account_finalizer.go

```diff
@@ -3,10 +3,25 @@ package account
 import (
 	"context"
 
+	"github.com/go-logr/logr"
 	awsv1alpha1 "github.com/openshift/aws-account-operator/pkg/apis/aws/v1alpha1"
 	"github.com/openshift/aws-account-operator/pkg/controller/utils"
 )
 
+// Function to add finalizer
```

Comment:
Probably it would be better to either have a comment that explains a bit more context about the function or not have a comment at all? This one doesn't tell me much more than reading the function name :)